microsoft-edge/webview2/reference/javascript/sharedbufferreceivedevent.yml

```diff
@@ -0,0 +1,131 @@
+### YamlMime:TSType
+name: SharedBufferReceivedEvent
+uid: 'WebView2Script!SharedBufferReceivedEvent:class'
+package: WebView2Script!
+fullName: SharedBufferReceivedEvent
+summary: >-
+  Event object for the `chrome.webview.sharedbufferreceived` event. This event is dispatched when
+  `CoreWebView2.PostSharedBufferToScript` is successfully called.
+remarks: |-
+
+
+  #### Examples
+
+  For Example, if we want to send data to script for one time read only consumption,
+
+  In native host app code, set data into the shared memory.
+
+  ```cpp
+  wil::com_ptr<ICoreWebView2ExperimentalEnvironment10> environment;
+  CHECK_FAILURE(
+      m_appWindow->GetWebViewEnvironment()->QueryInterface(IID_PPV_ARGS(&environment)));
+
+  wil::com_ptr<ICoreWebView2ExperimentalSharedBuffer> sharedBuffer;
+  CHECK_FAILURE(environment->CreateSharedBuffer(bufferSize, &sharedBuffer));
+  // Set data into the shared memory via IStream.
+  wil::com_ptr<IStream> stream;
+  CHECK_FAILURE(sharedBuffer->OpenStream(&stream));
+  CHECK_FAILURE(stream->Write(data, sizeof(data), nullptr));
+  PCWSTR additionalDataAsJson = L"{\"myBufferType\":\"bufferType1\"}";
+  if (fromFrame)
+  {
+      m_webviewFrame4->PostSharedBufferToScript(
+          sharedBuffer.get(), COREWEBVIEW2_SHARED_BUFFER_ACCESS_READ_ONLY,
+          additionalDataAsJson);
+  }
+  else
+  {
+      m_webView18->PostSharedBufferToScript(
+          sharedBuffer.get(), COREWEBVIEW2_SHARED_BUFFER_ACCESS_READ_ONLY,
+          additionalDataAsJson);
+  }
+  // Explicitly close the one time shared buffer to ensure that the resource is released.
+  sharedBuffer->Close();
+  ```
+
+  In the HTML document, subscribe to and handle sharedbufferreceived event.

```

Comment:
```suggestion
  In the HTML document, subscribe to and handle the `sharedbufferreceived` event.
```

Reply:
It's expected as part of the YAML indentation. I will keep the re-phrase and revert the indentation.

Reply:
```suggestion
  Next, in the HTML document, subscribe to and then handle the `sharedbufferreceived` event:
```